hack/util.sh

```diff
@@ -462,7 +462,7 @@ function kill_all_processes()
 
 	pids=($(jobs -pr))
 	for i in ${pids[@]-}; do
-		ps --ppid=${i} | xargs $sudo kill &> /dev/null
+		pgrep -P "${i}" | xargs $sudo kill &> /dev/null
```

Comment:
I am not for this change.

Especially in my more comprehensive changes to process cleanup, I use `ps`. Why bother with `pgrpep`?